cms/urls.py

```diff
@@ -186,8 +186,9 @@
 
 urlpatterns += (
     # Drops into the Programs authoring app, which handles its own routing.
-    # The view uses a configuration model to determine whether or not to
+    # These views use a configuration model to determine whether or not to
     # display the authoring app. If disabled, a 404 is returned.
+    url(r'^programs/id_token', ProgramsIdTokenView.as_view(), name='programs_id_token'),
```

Comment:
Since we expect this URL to be matched exactly, I'd use `r'^programs/id_token/$'`. I'd also move the comment on line 188 down so that it's directly above the URL which corresponds to the comment.